deploy/charts/impt/chart/Chart.yaml.template

```diff
@@ -0,0 +1,138 @@
+name: impt
+apiVersion: v2
+description: platform chart
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: ${TAG}
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: ${TAG}
+
+dependencies:
+  - name: opa
+    version: 0.0.5
+    condition: enabled.opa
+  - name: lgtm-stack
+    version: 0.0.3
+    condition: enabled.lgtm-stack
+  - name: oidc-stack
+    version: 0.0.2
+    condition: enabled.oidc-stack
+  - name: flyte-core
+    version: v0.0.1
+    condition: enabled.flyte-core
+  - name: web
+    version: ${TAG}
+    repository: "file://../../../../../web_ui/.build/chart"
+    condition: enabled.web
+  - name: impt-configuration
```

Comment:
if we no longer need these chart, please delete it
https://github.com/open-edge-platform/geti/tree/jchrapko/geti_installer/platform/services/config

Reply:
I switched to charts from the platform/services